### ISSUE TITLE
Gantry5 Admin Panel (dashboard) not using its css

### ISSUE THREAD

**Issue by Trust7** [gantry/gantry5#1395]
When I moved a website from local to live server the website is working fine but when I want to open the Gantry5 dashboard it shows up as below:

![gantry5dash_problem01](https://cloud.githubusercontent.com/assets/17771389/16296947/be2e454c-392e-11e6-86a9-66cad4691cce.jpg)

![gantry5dash_problem02](https://cloud.githubusercontent.com/assets/17771389/16296951/c32221ae-392e-11e6-96b8-2ee02ec42fb6.jpg)

I tried to move the website manually and with Akeeba Backup and it is the same. Any ideas?

**Comment by Trust7** [comment]
Ok, I have found the core of the issue. The problem was that it couldn't find g-admin.css and joomla-g-admin.css after the website transfer.

It happened because I made an override of 3 files in "templates/g5_hydrogen/custom/admin/templates/partials/". Those files are: base.html.twig, layout.html.twig, page.html.twig. I edited them and changed the path for g-admin.css and joomla-g-admin.css directing it to "custom" folder so I could change the Admin Panel appearance without changing G5 system files. Additionally I had my local website in a subfolder (not in root) and when I transfered the website the paths in those 3 files were not correct having this subfolder which doesn't exist in the live server.

Is there a better way to change the styles of the G5 admin panel than changing the paths in those 3 files to new CSS?
